code/game/gamemodes/miniantags/revenant/revenant_blight.dm

```diff
@@ -1,68 +0,0 @@
-/datum/disease/revblight
-	name = "Unnatural Wasting"
-	max_stages = 5
-	stage_prob = 10
-	spread_flags = NON_CONTAGIOUS
-	cure_text = "Holy water or extensive rest."
-	spread_text = "A burst of unholy energy"
-	cures = list("holywater")
-	cure_chance = 50 //higher chance to cure, because revenants are assholes
-	agent = "Unholy Forces"
-	viable_mobtypes = list(/mob/living/carbon/human)
-	disease_flags = CURABLE
-	permeability_mod = 1
-	severity = BIOHAZARD
-	var/stagedamage = 0 //Highest stage reached.
-	var/finalstage = 0 //Because we're spawning off the cure in the final stage, we need to check if we've done the final stage's effects.
-
-/datum/disease/revblight/cure()
-	if(affected_mob)
-		affected_mob << "<span class='notice'>You feel better.</span>"
-		if(affected_mob.dna && affected_mob.dna.species)
-			affected_mob.dna.species.handle_mutant_bodyparts(affected_mob)
-	..()
-
-/datum/disease/revblight/stage_act()
-	if(!finalstage && affected_mob.lying && prob(stage*5))
-		cure()
-		return
-	if(!finalstage && prob(stage*3))
-		affected_mob << "<span class='revennotice'>You suddenly feel [pick("sick and tired", "disoriented", "tired and confused", "nauseated", "faint", "dizzy")]...</span>"
-		affected_mob.confused += 10
-		affected_mob.adjustStaminaLoss(10)
-		PoolOrNew(/obj/effect/overlay/temp/revenant, affected_mob.loc)
-	if(!finalstage && stagedamage < stage)
-		stagedamage++
-		affected_mob.adjustToxLoss(stage*3) //should, normally, do about 45 toxin damage.
-		PoolOrNew(/obj/effect/overlay/temp/revenant, affected_mob.loc)
-	if(!finalstage && prob(45))
-		affected_mob.adjustStaminaLoss(stage*2)
-	..() //So we don't increase a stage before applying the stage damage.
-	switch(stage)
-		if(2)
-			if(prob(5))
-				affected_mob.emote("pale")
-		if(3)
-			if(prob(10))
-				affected_mob.emote(pick("pale","shiver"))
-		if(4)
-			if(prob(15))
-				affected_mob.emote(pick("pale","shiver","cries"))
-		if(5)
-			if(!finalstage)
-				finalstage = 1
-				affected_mob << "<span class='revenbignotice'>You feel like [pick("nothing's worth it anymore", "nobody ever needed your help", "nothing you did mattered", "everything you tried to do was worthless")].</span>"
```

Comment:
I'd be very sad if all this text went away, I like making the clown have a bad time.